modules/fbx/editor_scene_importer_fbx.cpp

```diff
@@ -107,6 +107,9 @@ Node3D *EditorSceneImporterFBX::import_scene(const String &p_path, uint32_t p_fl
 		f->get_buffer(data.ptrw(), data.size());
 		PackedByteArray fbx_header;
 		fbx_header.resize(64);
+
+		ERR_FAIL_COND_V(data.size() < 64, NULL);
```

Comment:
Maybe move it further up after the call to `f->get_buffer(data.ptrw(), data.size());`? No need to initialize and resize an array that we won't use if we abort.